apps/automation-v2/src/chains.ts

```diff
@@ -0,0 +1,61 @@
+export interface Chain {
+  id: number;
+  name: string;
+  subgraphUrl: string;
+  transactionUrl(txhash: string): string;
+  addressUrl(txhash: string): string;
+}
+
+export const MUMBAI: Chain = {
+  id: 80001,
+  name: 'Polygon Mumbai',
+  subgraphUrl: 'https://TODO',
+  transactionUrl(txhash: string) {
+    return `https://mumbai.polygonscan.com/tx/${txhash}`;
+  },
+  addressUrl(address: string) {
+    return `https://mumbai.polygonscan.com/address/${address}`;
+  },
+};
+
+export const ARBITRUM: Chain = {
+  id: 42161,
+  name: 'Arbitrum',
+  subgraphUrl: 'https://TODO',
+  transactionUrl(txhash: string) {
+    return `https://arbiscan.io/tx/${txhash}`;
+  },
+  addressUrl(address: string) {
+    return `https://arbiscan.io/address/${address}`;
+  },
+};
+
+export const MAINNET: Chain = {
+  id: 1,
+  name: 'Mainnet',
+  subgraphUrl: 'https://api.thegraph.com/subgraphs/name/medariox/v2-mainnet',
```

Comment:
I have a feeling we may have a different subgraph url for each 'thing', it won't be common for the whole chain.

ok for now, but it might be the case that this will have to move into the actual task config instead.